### PR TITLE
layer is compressed tar format

### PR DESCRIPTION
**Pull request by xiekeyang**
Signed-off-by: xiekeyang <xiekeyang@huawei.com>

### CODE REVIEW

File: config.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This specification uses the following terms:
 ### [Layer](layer.md)
 
 Image filesystems are composed of *layers*.
-Each layer represents a set of filesystem changes in a tar-based [layer format](layer.md), recording files to be added, changed, or deleted relative to its parent layer.
+Each layer represents a set of filesystem changes in a compressed tar-based [layer format](layer.md), recording files to be added, changed, or deleted relative to its parent layer.
```
Comment (wking) [inline config.md]: This is currently covered [here][1], and will no longer be true if/when #388 lands.  In any event, compression seems like a property of the later media type, and not a fundamental property of “being a layer”.

[1]: https://github.com/opencontainers/image-spec/blame/d1c7054fae33dd186c78fb126d5c613b233aa5eb/layer.md#L11

**Comment by xiekeyang** [comment]
@wking You are right, this should be closed.

**Comment by xiekeyang** [comment]
@wking I have a question.
> compression seems like a property of the later media type, and not a fundamental property of “being a layer”.

Then, if this comment are a little conflict to https://github.com/opencontainers/image-spec/pull/515, as `Do not confuse DiffIDs with layer digests, often referenced in the manifest, which are digests over compressed content.`?

If above words in #515 should be modified?

**Comment by wking** [comment]
> If above words in #515 should be modified?

Those words are fine now.  I'm [restoring][1] @stevvooe's "uncompressed" wording in #388.

[1]: https://github.com/opencontainers/image-spec/pull/515#discussion_r96984598

**Pull request closed without merging**